frontend/src/Album/AlbumCover.js

```diff
@@ -64,10 +64,9 @@ class AlbumCover extends Component {
       this.setState({
         cover: nextcover,
         coverUrl: getCoverUrl(nextcover, pixelRatio * size),
-        hasError: false
-        // Don't reset isLoaded, as we want to immediately try to
-        // show the new image, whether an image was shown previously
-        // or the placeholder was shown.
+        hasError: false,
+        isLoaded: true,
+        lazy: true
```

Comment:
is `lazy: true` needed here? it's not used on ArtistPoster or Banner, looks like default value is true.